### ISSUE TITLE
Warn about cache dir not found

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#216]
This doesn't give a warning when `/tmp/foobar` doesn't exist:
```
clj-kondo --lint src --cache /tmp/foobar
```

**Comment by borkdude** [comment]
It creates the directory `/tmp/foobar` when it doesn't exist. Discussed with @SevereOverfl0w that we can see this as a feature instead of a bug :-).
